### DISCUSSION TITLE
What will be the best POS system to integrate with medusa.js?

### DISCUSSION THREAD

**Discussion by edardev** [medusajs/medusa#4983] [Q&A]
I would love to see a native POS integration in medusa.js in the near future, for retailers and all SMBs, accept customer payments (cash, cards, digital wallets, etc.) Link customer accounts to get rewards, receipts, inventory, promotions and much more!

**Comment by uc77** [comment]
I'm building a native one that should be part of Medusa V2. Interested to join forces? Shoot us an email at unifiedcommercehq@gmail.com. More details here https://github.com/medusajs/medusa/discussions/9795
